File: common/src/main/java/com/ruzhan/common/util/ConstantUtils.java
package com.ruzhan.common.util;

import com.ruzhan.common.BuildConfig;

public final class ConstantUtils {

    public static final String TOKEN = "f32b30c2a289bfca2c9857ffc5871ac8";

    public static final String BUILD_TYPE_RELEASE = "release";
    public static final String BUILD_TYPE_BETA = "beta";
    public static final String BUILD_TYPE_DEBUG = "debug";

    public static final int PAGE_SIZE = 10;

    public static boolean isReleaseBuild() {
        return BuildConfig.BUILD_TYPE.equals(BUILD_TYPE_RELEASE);
    }

    public static boolean isBetaBuild() {
        return BuildConfig.BUILD_TYPE.equals(BUILD_TYPE_BETA);
    }

    public static boolean isDebugBuild() {
        return BuildConfig.BUILD_TYPE.equals(BUILD_TYPE_DEBUG);
    }
}

File: common/src/main/java/com/ruzhan/common/util/ConstantUtils.kt
package com.ruzhan.common.util

import com.ruzhan.common.BuildConfig

object ConstantUtils {

    const val TOKEN = "f32b30c2a289bfca2c9857ffc5871ac8"

    private const val BUILD_TYPE_RELEASE = "release"
    private const val BUILD_TYPE_BETA = "beta"
    private const val BUILD_TYPE_DEBUG = "debug"

    const val PAGE_SIZE = 10

    @JvmStatic
    val isReleaseBuild: Boolean
        get() = BuildConfig.BUILD_TYPE == BUILD_TYPE_RELEASE

    @JvmStatic
    val isBetaBuild: Boolean
        get() = BuildConfig.BUILD_TYPE == BUILD_TYPE_BETA

    @JvmStatic
    val isDebugBuild: Boolean
        get() = BuildConfig.BUILD_TYPE == BUILD_TYPE_DEBUG
}
